bbc/simorgh
# Use updated Pashto locale to introduce relative time support

Resolves https://github.com/bbc/simorgh/issues/3147

**Overall change:** Use the latest @bbc/psammead-locales dependency, which contains an updated Pashto locale that supports relative timestamps.

**Code changes:**

- Update `@bbc/psammead-locales` entry in package.json to the latest version.
- Add Pashto locale to Moment Locales story. http://localhost.bbc.com:9001/?path=/story/moment-locales-editorial-view--pashto-ps

**Manual testing process:**
1. Generate a timestamp using https://currentmillis.com/ for:
    - One minute since epoch.
    - Several minutes since epoch.
    - One hour since epoch.
    - Several hours since epoch.
    - More than 10 hours since epoch.
    - More than a day since epoch.
2. Change `firstPublished` and `lastPublished` values in `/data/pashto/articles/cng0e8v85eko.json` to verify the generated timestamps are displayed correctly in the application: http://localhost.bbc.com:7080/pashto/articles/cng0e8v85eko.
3. Verify the Pashto locale is displaying correctly in Storybook: http://localhost.bbc.com:9001/?path=/story/moment-locales-editorial-view--pashto-ps.

**Manual testing results:**

**Simorgh Article:**
One minute since epoch
![one-minute](https://user-images.githubusercontent.com/1430787/65591020-aac11080-df83-11e9-9d43-6933b23d0a18.png)

Several minutes since epoch
![several-minutes](https://user-images.githubusercontent.com/1430787/65591026-ad236a80-df83-11e9-94e2-47c83e933bcc.png)

One hour since epoch
![one-hour](https://user-images.githubusercontent.com/1430787/65591032-b01e5b00-df83-11e9-9c5f-8994a08de766.png)

Several hours since epoch
![several-hours](https://user-images.githubusercontent.com/1430787/65591042-b1e81e80-df83-11e9-8a34-524ed02b7223.png)

More than 10 hours since epoch
![more-than-ten-hours](https://user-images.githubusercontent.com/1430787/65591053-b44a7880-df83-11e9-91d0-4c52e62d123a.png)

More than a day since epoch
![absolute](https://user-images.githubusercontent.com/1430787/65591060-b7456900-df83-11e9-8b60-3051646f04ad.png)

**Storybook:**
![after](https://user-images.githubusercontent.com/1430787/65591008-a4cb2f80-df83-11e9-9407-bc834a0790f1.png)

---
